Makefile

```diff
@@ -198,17 +200,17 @@ kafka:
 ifeq ($(OLM),true)
 	@echo Skipping kafka-operator deployment, assuming it has been installed via OperatorHub
 else
-	@sed 's/namespace: .*/namespace: kafka/' ./test/kafka-operator.yml | kubectl -n $(KAFKA_NAMESPACE) apply -f -  2>&1 | grep -v "already exists" || true
+	@curl --location $(KAFKA_YAML) |  sed 's/namespace: .*/namespace: $(KAFKA_NAMESPACE)/' | kubectl -n $(KAFKA_NAMESPACE) apply -f -  2>&1 | grep -v "already exists" || true
 endif
-	@kubectl apply -f ./test/kafka.yml -n $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
+	@curl --location $(KAFKA_EXAMPLE) | kubectl -n $(KAFKA_NAMESPACE) apply -f -  2>&1 | grep -v "already exists" || true
 
 .PHONY: undeploy-kafka
 undeploy-kafka:
-	@kubectl delete -f ./test/kafka.yml -n $(KAFKA_NAMESPACE) 2>&1 || true
+	@kubectl get kafkas --namespace $(KAFKA_NAMESPACE) --output jsonpath='{.items[0].metadata.name}' | xargs kubectl delete kafka --namespace $(KAFKA_NAMESPACE)
 ifeq ($(OLM),true)
 	@echo Skiping kafka-operator undeploy
 else
-	@kubectl delete -f ./test/kafka-operator.yml 2>&1 || true
+	@kubectl get deployments --namespace $(KAFKA_NAMESPACE) --output jsonpath='{.items[0].metadata.name}' | xargs kubectl delete deployment --namespace $(KAFKA_NAMESPACE)
```

Comment:
I'm not sure if that will remove the CRDs, but I can experiment.  In the original version of this PR I was downloading the CRs (with curl --output) but someone (maybe @jpkrohling) suggested I do curl and pipe directly to kubectl apply instead.

What do you think is the best way to do this?

Reply:
IMO, the best would be to have the CRs in our repository (no `curl -o` nor `curl | kubectl`). Our instructions may contain `curl | kubectl`, but for I'd have a static resource for CI, to avoid getting unrelated failures in our builds. But between downloading with `curl -o` and directing the output directly to `kubectl`, I'd go with the option that doesn't store a file in disk.

About the command: it will not remove CRDs, but our `clean` target should leave the cluster in a state that is close to the beginning, ie, without CRDs, roles, role bindings, operators, operands, accounts, deployments, ... Deleting the operator might cascade the deletion of operands, but CRDs/roles/role bindings probably have to be removed manually.

Reply:
By the way: the same way you can pipe the contents from curl to `kubectl apply -f -`, you can pipe it to `kubectl delete -f -`.

Reply:
@jpkrohling Sorry, I should have made it more clear, but one of the primary reasons for this change is to permit testing using CRs from AMQ Streams, which of course would come from an entirely different repo.  So I'm trying to find the best solution for that situation.

Reply:
We discussed this on IRC, and the current state LGTM: the files are downloaded to a location that is part of .gitignore. This way, network failures are avoided when performing the delete operation.